packages/react-components/src/InputBalance.tsx

```diff
@@ -35,7 +35,8 @@ interface Props extends BareProps {
 const DEFAULT_BITLENGTH = BitLengthOption.CHAIN_SPEC as BitLength;
 
 function InputBalance ({ autoFocus, className, defaultValue, help, isDisabled, isError, isFull, isZeroable, label, labelExtra, maxValue, onChange, onEnter, onEscape, placeholder, style, value, withEllipsis, withLabel, withMax }: Props): React.ReactElement<Props> {
-  const formattedDefaultValue =  defaultValue ? toFormattedBalance({ value: defaultValue }) : '0';
+  const formattedDefaultValue =  defaultValue ? toFormattedBalance({ value: defaultValue }) : "";
```

Comment:
add some better defaults
amount can be nothing "" and placeholder default is `0.0`